lib/core/src/lib/app-config/app-config.service.ts

```diff
@@ -192,8 +192,9 @@ export class AppConfigService {
                 this.http.get(configUrl).subscribe(
                     (data: any) => {
                         this.status = Status.LOADED;
+                        resolve(data);
+                        // WARNING: Risky change! Despite the fact that this would be the right order, this is a breaking change...
```

Comment:
We don't need this warning anymore, we already know we are doing breaking changes.